CHANGELOG.md

```diff
@@ -13,16 +13,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `#[should_panic]` attribute support
 - Documentation to public methods
+- Information sections to documentation about importing `snforge_std`
 
 #### Changed
 
-- Exported cheatcodes as a Scarb package, now you have to import them explicitly with e.g. `use cheatcodes::declare`
-  and add them as a dependency to your Scarb.toml
+- Exported snforge_std as a Scarb package, now you have to import it explicitly with e.g. `use snforge_std::declare`
```

Comment:
Just a nitpick but sometimes we use snforge_std and other times we use `snforge_std`

Reply:
It is a changelog so it is not a big deal I think, but will change it